compiler/apps/playground/__tests__/e2e/__snapshots__/page.spec.ts/parse-flow-output.txt

```diff
@@ -0,0 +1,14 @@
+import { c as _c } from "react/compiler-runtime";
+function useFoo(propVal) {
+  const $ = _c(2);
+  const t0 = (propVal.baz: number);
```

Comment:
Flow now codegens and prints successfully. This means that Flow users can use playground without needing to edit / manually strip their annotations